Gopkg.toml

```diff
@@ -15,10 +15,28 @@
 #   branch = "dev"
 #   source = "github.com/myfork/project2"
 #
-# [[override]]
+# [[constraint]]
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+required = [
+  "k8s.io/api/core/v1",
+  "k8s.io/apimachinery/pkg/api/errors",
+  "k8s.io/apimachinery/pkg/api/meta",
+  "k8s.io/apimachinery/pkg/apis/meta/v1",
+  "k8s.io/apimachinery/pkg/watch",
+  "k8s.io/client-go/kubernetes",
+  "k8s.io/client-go/rest",
```

Comment:
I think it's enough to have one (random) line per imported dependency. Also dependencies which you have imported in your code don't need to be there (i.e. k8s.io/client-go).